test/test_macho.rb

```diff
@@ -313,8 +321,15 @@ def test_change_install_name
 
       file.write(actual)
 
-      # compare actual and expected file hashes, to ensure file correctness
-      assert equal_sha1_hashes(actual, expected)
```

Comment:
Yeah, bit-for-bit equality is probably preferable to sha1 hashes (even if the chance of a hash collision is vanishingly small). Will fix :+1: